internal/pkg/aws/sessions/sessions.go

```diff
@@ -169,3 +179,7 @@ func userAgentHandler() request.NamedHandler {
 		},
 	}
 }
+
+func isRegionMissing(sess *session.Session) bool {
+	return sess.Config.Region == nil || *sess.Config.Region == ""
```

Comment:
```suggestion
	return aws.StringValue(sess.Config.Region) == ""
```
Maybe we don't need a function for this logic?